app/views/users/new.html.erb

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>New</title>
+</head>
+<body>
+  <%# <h1>Hey!</h1> %>
+  <%# <form accept-charset="UTF-8" action="/users" method="post"> 
+    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+    
+    <%# <label for="username">Username:</label>
+    <input id="username"  name="user[username]" type="text"><br>
+
+    <label for="email">e-mail:</label>
+    <input id="email" name="user[email]" type="text"><br>
+
+    <label for="password">Password:</label>
+    <input id="password" name="user[password]" type="password"><br>
+
+    <input type="submit" value="Submit"> 
+
+  </form>%>
+  
+  <!-- <%= form_tag("/users", method: "post") do %>
```

Comment:
- [ ] Use `form_tag(users_path)` without the `method: "post"`. This should work as well.